nested/nfs-ha.json

```diff
@@ -148,7 +148,7 @@
             "sku": {
                 "name": "Basic"
             },
-            "apiVersion": "2017-10-01",
+            "apiVersion": "latest",
```

Comment:
Let's not use "latest" for apiVersion, let's have a unique date string around this commit/time-frame for every applicable mention of apiVersion.